pegasus/sites.v3/hourofcode.com/styles/050-theme.css

```diff
@@ -80,7 +80,7 @@ body {
 #pagefooter {
   width: 100%;
   padding: 2em 2.5em 2.5em;
-  background-color: #5c3f7f;
+  background-color: var(--neutral_dark);
```

Comment:
Changed the footer color so it doesn't clash with the new purple buttons:

### Before
<img width="967" alt="Screenshot 2023-10-05 at 8 56 38 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/db11c481-d8bd-4369-86d5-90fa0e0f8d83">

### After
<img width="969" alt="Screenshot 2023-10-05 at 8 56 16 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/b6a9e8ff-f65a-49f5-b93c-0243c09ca731">